packaging/rpm/microshift.spec

```
@@ -221,8 +223,12 @@ fi
 /var/run/secrets/kubernetes.io/serviceaccount
 %{_datadir}/selinux/packages/%{selinuxtype}/microshift.pp.bz2
 %ghost %{_sharedstatedir}/selinux/%{selinuxtype}/active/modules/200/microshift
+%{_unitdir}/hostpath-provisioner.service
 
 %changelog
+* Tue May 24 2022 Ricardo Noriega <rnoriega@redhat.com> . 4.10.0-0.microshift-2022-04-23-131357
```

```suggestion
* Tue May 24 2022 Ricardo Noriega <rnoriega@redhat.com> . 4.10.0-0.microshift-2022-04-23-131357_2
```

That comes from the Version & Release parameters that we auto-generate, in fact it will be the same
as you wrote, so what you wrote *is correct*, but I suspect some rpm building tools will complain
if we have two changelogs for the same release, so just in case....

---

+1